internal/provider/kubernetes/kubernetes.go

```diff
@@ -1,3 +1,7 @@
+// Copyright 2022 Envoy Gateway Authors
```

Comment:
Should we remove the year from the boilerplate so we don't have to maintain it?

xref: https://github.com/kubernetes-sigs/gateway-api/commit/afbd2cc7db6cf4a08e9b3bc969baca9f83a85171